Source/Data/Symbols.txt

```diff
@@ -1141,4 +1138,8 @@
 🧠 ㄋㄠˇ -8
 🧎‍♀️ ㄍㄨㄟˋ -8
 🧎 ㄍㄨㄟˋ -8
-🧎‍♂️ ㄍㄨㄟˋ -8
\ No newline at end of file
+🧎‍♂️ ㄍㄨㄟˋ -8
+⚠️ ㄐㄧㄥˇ-ㄍㄠˋ -8
+🪼 ㄕㄨㄟˇ-ㄇㄨˇ -8
```

Comment:
macos 12.7 can't see it.

Reply:
![image](https://github.com/openvanilla/McBopomofo/assets/86348833/a5ff3802-3111-48df-afeb-ca9dab9c064e)
Can both be seen on iOS 16.5 with JoyPixel Decal
And Arch Linux appleemoji